cinema/migrations/0002_movie_image.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.1 on 2024-12-11 23:13
+
+import cinema.models
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cinema', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='movie',
+            name='image',
+            field=models.ImageField(blank=True, null=True, upload_to=cinema.models.movie_image_file_path),
```

Comment:
Ensure that the `movie_image_file_path` function in `cinema.models` is defined to format the image filename as specified: `f"{slugify(movie.title)}-{uuid}{ext}"`. This is crucial for meeting the task requirements regarding image file naming.